FileShares/src/FileSharesWeb/Views/Files/Upload.cshtml

```diff
@@ -0,0 +1,38 @@
+@model Dictionary<string, string>?
+@{
+    ViewData["Title"] = "Upload Files";
+}
+
+@if (Model?.Count > 0)
+{
+    if (Model.Any(file => file.Key == "error"))
```

Comment:
Why such a hacky solution by stashing the error in a dictionary? Should have a proper model with `.Error` and `.Files`, both nullable.